guides/advanced_guides/relevancy.md

```diff
@@ -0,0 +1,204 @@
+
+The `attribute` rule sorts by ascending [attribute importance](/guides/advanced_guides/relevancy.md#attributes-importance).
+
+:::
+
+::: tab Words position
+
+![Image from alias](../../public/image/belgium.png)
+
+### Word position
+
+`Gangsta` appears sooner than `Dunkirk` because `Belgium` appears sooner in the attribute.
+
+The `word position` rule orders by ascending matching word's index number.
+
+:::
+
+::: tab Exactness
+![Image from alias](../../public/image/knight.png)
+
+**Exactness**
+
+`Knight Moves` is displayed before `Knights of Badassdom` because `Knight` is a finished word, exactly the same as the search query. While with `Knights`, the search query is just a prefix.
+
+:::
+
+::::
+
+## Order of the rules
+
+Depending on your needs, you might want to change the order in which the rules are processed.
+
+In this case, using the [settings route](/references/ranking_rules.md#add-or-replace-ranking-rules) of your index, you can change the ranking order of the sorting rules.
+
+```json
+[
+  "_typo",
+  "_attribute",
+  "_proximity",
+  "_words",
+  "_words_position",
+  "_exact"
+]
+```
+
+## Adding your rules
+
+New rules can be added to the existing list at any time and anywhere in the existing order.
+
+A custom rule lets you create a descending or ascending sorting rule on a given attribute.
+To create a rule, you will need to communicate the attribute on which the rule is created and the order in which it will be sorted: `Asc(attribute_name)` or `Desc(attribute_name)`.
+
+This rule must be added to the existing list of ranking rules using the [settings route](/references/ranking_rules.md#add-or-replace-ranking-rules).
+
+#### Example
+
+```
+Desc(release_date)
+```
+This will create a rule that makes recent movies more relevant than older ones.
+
+```
+Asc(movie_ranking)
+```
+This will create a rule that makes movies with a good rank more relevant than others.
+
+To add this newly created rule to the existing ranking rule, using the [settings route](/references/ranking_rules.md#add-or-replace-ranking-rules), you need to add the rule in the existing order array.
+
+```json
+[
+  "_typo",
+  "_attribute",
+  "_proximity",
+  "_words",
+  "_words_position",
+  "_exact",
+  "Desc(release_date)",
+  "Asc(movie_ranking)"
+]
+```
+
+## Attributes importance
+
+In a dataset, some fields are more relevant to the search than others. A `title`, for example, has a value more meaningful to a movie search than its `description` or its `director` name.
+
+By default, MeiliSearch will determine the order of importance of the attributes based on the order in which they appear in the first document added. Then, each new attribute present in new documents will be added to the end of this ordered list.
+
+This means that, for this inference to work, the order of the fields in your dataset must be ordered before they are added to MeiliSearch. Or at least the first document.
```

Comment:
I bugged on "for this inference to work", I mean I had to read this sentence multiple time. But maybe it's the right way to turn the sentence in English.

Reply:
Maybe if we remove the "This means that" And just start the sentence with "For this inference to work". What do you think?